.github/workflows/nodejs.yml

```diff
@@ -3,10 +3,12 @@ name: Node.js CI
 on:
   push:
     branches:
-      - master
+      - main
```

Comment:
The default branch is `master` here. We shouldn't make this change before renaming it.

Reply:
Thanks. Maybe we should update documentation to Typedoc action as well.